UPGRADE-6.0.md

```diff
@@ -146,6 +146,15 @@ Routing
  * Removed `RouteCollectionBuilder`.
  * Added argument `$priority` to `RouteCollection::add()`
  * Removed the `RouteCompiler::REGEX_DELIMITER` constant
+ * `Route` and `CompiledRoute` do not implement the deprecated `Serializable`
+   interface anymore. The consequence is that objects that were serialized with
+   PHP 7.3 or below cannot be unserialized anymore. If an application stores
+   serialized instances of those classes, those have to be regenerated before
+   upgrading to Symfony 6. This can be done by running the following code on
+   PHP 7.4 or newer:
```

Comment:
I would explicitly add in the sentence that this has to be run using symfony/routing 4.4 or 5.x (version range to be checked for 4.4 and 5.x to check which versions have the support for both serialization methods as I'm not sure `__serialize` was already there in 4.4.0 and 5.0), to avoid confusion.